### ISSUE TITLE
Problem with package: smsmobileapi

### ISSUE THREAD

**Issue by SmsMobileApi** [piwheels/packages#483]
### Package name

smsmobileapi

### Package version

1.0.8

### PyPI URL

https://pypi.org/project/smsmobileapi/

### piwheels URL

https://www.piwheels.org/project/smsmobileapi/

### Python version

- [X] Python 3.9
- [ ] Python 3.11

### I am aware this is the issue tracker for a Python package index specifically for Raspberry Pi

- [ ] Yes

### I have checked for duplicate issues

- [ ] Yes

### I am the maintainer

- [X] Yes

### More information

Hello,
I'm the developer of the SMS Mobile API Python module.
Can you tell me why your website reports my module as "unstable"? When I check the webpage presenting my module, it seems impossible to install.
Thanks for your feedback.

**Comment by bennuttall** [comment]
I don't know where you got the word "unstable" from?

piwheels aims to build wheels for all packages on PyPI. The builds failed for this package, which is why the red cross icon is shown. If you click one of the red cross icons it will show you the build output for that build.

For example: https://www.piwheels.org/logs/0000/1248/8780.txt

This shows that pip failed to generate the metadata for your package:

```
pip._internal.exceptions.MetadataGenerationFailed: metadata generation failed
```

You may wish to investigate, fix the problem in your `setup.py` (or move to `pyproject,toml`) and make a new release.

Note that these failing builds are not preventing anyone from installing your package. Users on a Raspberry Pi will still be able to install the wheel you provided on PyPI.

**Comment by SmsMobileApi** [comment]
Yes you are right. I used the term unsable  but I just sax the red cross to the webpage.

Il will fix my setup.py to get a Green cross of your webpage ;-)

I didnt know your webspage before but it's look like a good tool for dev ;-)

When I fix the setup.py have I have to contact you to run a test or it's a automatic process ? 

⁣￼

Thomas B.
CEO - SMS Mobile API
Simplify your communications with our messaging API.

Connect with us:
￼   ￼   ￼​

Le 20 oct. 2024, 13:38, à 13:38, Ben Nuttall ***@***.***> a écrit:

**Comment by bennuttall** [comment]
When you do a new release (a new version) it will automatically be built.

**Comment by SmsMobileApi** [comment]
Ok great


⁣

Thomas B.
CEO - SMS Mobile API
Simplify your communications with our messaging API.

Connect with us:
      ​

Le 20 oct. 2024, 16:17, à 16:17, Ben Nuttall ***@***.***> a écrit:

**Comment by MichaIng** [comment]
All 2.0.1 wheels are present, hence closing this.
